svf/include/SABER/SaberSVFGBuilder.h

```diff
@@ -97,6 +101,8 @@ class SaberSVFGBuilder : public SVFGBuilder
     PointsTo globs;
     /// Store all global SVFG nodes
     SVFGNodeSet globSVFGNodes;
+    /// Removed strong update edge
+    Map<const SVFGNode*, SVFGNodeSet> removedSUVFEdges;
```

Comment:
better to move this map to SaberCondAllocator

Reply:
I'm not sure. removedSUVFEdges is derived from saber svfg builder. The Sabercondallocator is not yet initialized at the moment.